llama_stack/distribution/server/auth_providers.py

```diff
@@ -347,6 +475,8 @@ def create_auth_provider(config: AuthenticationConfig) -> AuthProvider:
         return CustomAuthProvider(CustomAuthProviderConfig.model_validate(config.config))
     elif provider_type == "oauth2_token":
         return OAuth2TokenAuthProvider(OAuth2TokenAuthProviderConfig.model_validate(config.config))
+    elif provider_type == "openshift_oauth":
```

Comment:
Again, the 'oauth' here is in my view misleading. This is simply an openshift specific approach to token verification is it not?

Reply:
that's correct. I can rename it.